app/templates/components/fastboot-only/body-bottom.hbs

```diff
@@ -7,3 +7,5 @@
 {{/if}}
 
 {{fastboot-only/local-navigation-for-seo}}
+
+<script type="x-head-data-store" id="head-data-store">{{{data}}}</script>
```

Comment:
I know that shoebox probably do similar thing but are we sure `data` is safe? No XSS vulnerability?